luzer/luzer.c

```diff
@@ -327,57 +328,10 @@ load_custom_mutator_lib(void) {
 NO_SANITIZE static int
 luaL_fuzz(lua_State *L)
 {
-	if (lua_istable(L, -1) == 0) {
-		luaL_error(L, "opts is not a table");
-	}
-	lua_pushnil(L);
```

Comment:
Looks as non-relevant change

Reply:
This check is needed if we expect a table with optionals, so I decided to move it to the luser_args.c file.